source/jormungandr/jormungandr/street_network/asgard.py

```diff
@@ -54,35 +54,16 @@
 # https://valhalla.readthedocs.io/en/latest/api/turn-by-turn/api-reference/#supported-language-tags
 # Be careful, the syntax has to be exact
 class Languages(Enum):
-    bulgarian = "bg-BG"
-    catalan = "ca-ES"
-    czech = "cs-CZ"
-    danish = "da-DK"
-    german = "de-DE"
-    greek = "el-GR"
-    english_gb = "en-GB"
-    english_pirate = "en-US-x-pirate"
-    english_us = "en-US"
-    spanish = "es-ES"
-    estonian = "et-EE"
-    finnish = "fi-FI"
+    dutch = "nl-NL"
+    english = "en-GB"
```

Comment:
Maybe `en-US` is a more sensible default? It's usually the default in a lot of systems, and we usually tend to speak/translate to a US variant, more than to a GB variant.

Note that if you agree to use `en-US` as a default value, there might a lot of places in your PR to modify too (in order to make it the default everywhere).

Reply:
:+1: